### PR TITLE
Removed outdated TODOs

### PR DESCRIPTION
**Pull request by marcusnaslund**
Cleaned out TODOs that aren't useful, outdated, or simply won't be fixed.

### CODE REVIEW

File: source/base/Event.ooc [hunk 1]
```diff
@@ -83,7 +83,7 @@ Event1: class <T> {
 	}
 }
 
-Event2: class <T0, T1> { // TODO: Write tests and fix this
```
Comment (marcusnaslund) [inline source/base/Event.ooc]: I just don't know what's left to fix.

File: source/io/CsvReader.ooc [hunk 2]
```diff
@@ -2,8 +2,6 @@ use ooc-base
 use ooc-collections
 import io/[File, FileReader]
 
-// TODO: Handle multi-line string literals
```
Comment (marcusnaslund) [inline source/io/CsvReader.ooc]: Discussed in October that we have no use for this atm https://github.com/cogneco/ooc-kean/issues/566

File: test/math/FloatComplexTest.ooc [hunk 3]
```diff
@@ -61,7 +61,6 @@ FloatComplexTest: class extends Fixture {
 			expect((complexNumber2 / complexNumber0) real, is equal to(2.6f) within(tolerance))
 			expect((complexNumber2 / complexNumber0) imaginary, is equal to(0.2f) within(tolerance))
 		})
-		// TODO: Add tests for compound assignment operators
```
Comment (marcusnaslund) [inline test/math/FloatComplexTest.ooc]: We don't test operators for most classes/covers, which is a problem, but it's not specific to `FloatComplex`. I will create a more general issue.

**Comment by marcusnaslund** [comment]
Comments @sebastianbaginski ?

**Comment by sebastianbaginski** [comment]
No, I have nothing to add

**Pull request merged**